plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -621,4 +624,11 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public Image getImage() {
+    ImageDescriptor imageDescriptor = AbstractUIPlugin
+        .imageDescriptorFromPlugin(DataflowUiPlugin.PLUGIN_ID, "icons/Dataflow_16.png");
+    return imageDescriptor != null ? imageDescriptor.createImage() : null;
```

Comment:
We need to track this image and dispose of it in `dispose()`, otherwise we'll leak images.